sgx-hasher/src/lib.rs

```diff
@@ -77,6 +81,68 @@ impl Hasher {
     }
 }
 
+pub fn to_array(bn: bn::BigNum) -> [u8; 384] {
```

Comment:
This function probably shouldn't be `pub`. It also appears to ignore endianness issues. The array is little endian but openssl is big endian.